package/gluon-core/files/lib/netifd/proto/gluon_wired.sh

```diff
@@ -56,6 +77,7 @@ proto_gluon_wired_setup() {
         json_init
         json_add_string name "${config}_mesh"
         json_add_string ifname "@${meshif}"
+        [ -n "$index" ] && json_add_string macaddr "$(lua -e "print(require('gluon.util').generate_mac($index))")"
```

Comment:
Remove this, it should be meaningless in VXLAN mode (as the MAC address is set above), and incorrect otherwise (will modify the MAC address of an underlying interface, but we already set all MAC addresses as intended).